aidants_connect_web/templates/aidants_connect_web/new_mandat/new_mandat.html

```diff
@@ -44,11 +44,14 @@ <h2>Mentions à lire à l'usager</h2>
     <div class="container">
       <form method="post">
         {% if form.errors %}
-          <div class="notification error" role="alert">{{ form.errors }}</div>
+        <div class="notification error" role="alert">Il y a des erreurs dans le formulaire</div>
```

Comment:
Voir [r612566098](https://github.com/betagouv/Aidants_Connect/pull/343/files#r612566098).

Reply:
Très cool d'afficher les messages au bon endroit. (Team points à la fin des phrases, toutefois)
```suggestion
        <div class="notification error" role="alert">Il y a des erreurs dans le formulaire.</div>
```